packages/terra-consumer-nav/src/components/nav-help/NavHelp.scss

```diff
@@ -0,0 +1,129 @@
+@import '../../variables';
+
```

Comment:
make sure to run the linters, i'm seeing quite a few here that will throw red-flags